static-analysis/autodispose-lint/src/main/kotlin/com/uber/autodispose/lint/AutoDisposeDetector.kt

```diff
@@ -46,7 +46,11 @@ class AutoDisposeDetector: Detector(), SourceCodeScanner {
         Category.CORRECTNESS,
         10,
         Severity.ERROR,
-        Implementation(AutoDisposeDetector::class.java, EnumSet.of(Scope.JAVA_FILE)))
+        Implementation(AutoDisposeDetector::class.java,
```

Comment:
Before merging, can you add a comment explaining why we have to do it this way?

Reply:
b7df26136f0822c527f1de85cb0b251a26c8fad3